### PR TITLE
demo(accordion): add basic accordion demo, clean previous ex

### PR DESCRIPTION
**Pull request by ExFlo**
I add the same basic accordion demo as in bootstrap website, and one demo of the closeOthers feature with also static panel.

### CODE REVIEW

**Comment by Foxandxss** [comment]
Ah, that is something I had in my list too :/

Will review soon.

**Comment by ExFlo** [comment]
addresses cleaning of accordion portion of #202

**Comment by icfantv** [comment]
LGTM.  @pkozlowski-opensource?

File: demo/src/app/components/accordion/demos/basic/basic.component.html [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
-  <button type="button" class="btn btn-primary btn-sm" (click)="firstDisabled = !firstDisabled">Enable / Disable first panel</button>
-  <button type="button" class="btn btn-danger btn-sm" (click)="removeDynamic()">Remove last dynamic</button>
-</p>
-
-<ngb-accordion close-others="true">
-  <ngb-panel title="This is the header" [disabled]="firstDisabled" open="true">
-    This is the content
+<ngb-accordion>
+  <ngb-panel title="Collapsible Group Item #1" open="true">
+    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf moon tempor, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident. Ad vegan excepteur butcher vice lomo. Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably haven't heard of them accusamus labore sustainable VHS.
```
Comment (icfantv) [inline demo/src/app/components/accordion/demos/basic/basic.component.html]: just a nit, would you mind wrapping these at like 120 or so?  i'd like the demo HTML to fit without scrolling if entirely possible.

File: demo/src/app/components/accordion/demos/basic/basic.component.html [hunk 2]
```diff
@@ -1,17 +1,11 @@
...
   </ngb-panel>
-  <ngb-panel [title]="group.heading" *ngFor="let group of groups">
-    {{group.content}}
+  <ngb-panel title="Collapsible Group Item #2">
+    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf moon tempor, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident. Ad vegan excepteur butcher vice lomo. Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably haven't heard of them accusamus labore sustainable VHS.
```
Comment (icfantv) [inline demo/src/app/components/accordion/demos/basic/basic.component.html]: ditto

File: demo/src/app/components/accordion/demos/basic/basic.component.html [hunk 3]
```diff
@@ -1,17 +1,11 @@
...
   </ngb-panel>
-  <ngb-panel title="Another group" [open]="isOpen">
-    More content
+  <ngb-panel title="Collapsible Group Item #3">
+    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf moon tempor, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident. Ad vegan excepteur butcher vice lomo. Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably haven't heard of them accusamus labore sustainable VHS.
```
Comment (icfantv) [inline demo/src/app/components/accordion/demos/basic/basic.component.html]: ditto.

File: demo/src/app/components/accordion/accordion.component.html [hunk 4]
```diff
@@ -1,7 +1,10 @@
 <ngbd-content-wrapper component="Accordion">
   <ngbd-api-docs directive="NgbAccordion"></ngbd-api-docs>
+  <ngbd-example-box demoTitle="Basic demo close others" [htmlSnippet]="staticHtmlContent" [tsSnippet]="staticTsContent">
+    <ngbd-accordion-static></ngbd-accordion-static>
+  </ngbd-example-box>
   <ngbd-api-docs directive="NgbPanel"></ngbd-api-docs>
-  <ngbd-example-box demoTitle="Basic demo" [htmlSnippet]="basicHtmlContent" [tsSnippet]="basicTsContent">
+  <ngbd-example-box demoTitle="Basic demo Panel" [htmlSnippet]="basicHtmlContent" [tsSnippet]="basicTsContent">
```
Comment (icfantv) [inline demo/src/app/components/accordion/accordion.component.html]: Basic Demo Panel

**Comment by pkozlowski-opensource** [comment]
@ExFlo it looks like something went wrong with the git operations. Could you please rebase and make sure that there is just one commit in this PR?

**Comment by pkozlowski-opensource** [comment]
@ExFlo it is better (that is, we don't have merge commit) but it is not rebased on top of the current master (so doesn't merge cleanly). You need to:
* pull the latest master from upstream
* rebase your branch on top of the master
* force push o GH

**Pull request closed without merging**